app/javascript/gobierto_data/webapp/components/sidebar/SidebarDatasets.vue

```diff
@@ -1,65 +1,50 @@
 <template>
-  <div>
+  <div v-if="listDatasets.length">
```

Comment:
If you don't need a `div`, only to set the `v-if`, try better:
```suggestion
  <template v-if="listDatasets.length">
```

Reply:
You can't use a template here because there's not a single child but multiple in the same level.